### PR TITLE
Closing unreleased/open streams in certain locations

### LINKED ISSUE #1527: KeyStoreKeyFactory: InputStream not being closed
Running out of file handles

### PR DESCRIPTION
**Pull request by sourabhsparkala**
Fixes #1527

- Some open or unreleased streams were closed at certain locations. These locations were identified by a Static code analysis tool. 
- Formatted as per already existing file's original format 
- Added loggers to indicate if there are any issues while trying to close the stream

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@sourabhsparkala Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-security-oauth&pullRequestId=1708)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-security-oauth&pullRequestId=1708) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@sourabhsparkala Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-security-oauth&pullRequestId=1708)!

File: spring-security-oauth/src/main/java/org/springframework/security/oauth/config/ConsumerDetailsFactoryBean.java [hunk 1]
```diff
@@ -33,7 +36,8 @@
  * 
  */
 public class ConsumerDetailsFactoryBean implements FactoryBean<ConsumerDetails>, ResourceLoaderAware {
-
+	
+	private static Log logger = LogFactory.getLog(ConsumerDetailsFactoryBean.class);
```
Comment (artem-smotrakov) [inline spring-security-oauth/src/main/java/org/springframework/security/oauth/config/ConsumerDetailsFactoryBean.java]: `logger` may be final.

**Comment by jgrandja** [comment]
Thanks for the PR @sourabhsparkala. This is now in master.

**Pull request closed without merging**